packages/client-sdk-web/src/internal/data-client.ts

```diff
@@ -1082,4 +1650,37 @@ export class DataClient<
   private convertArrayToUint8(v: (string | Uint8Array)[]): Uint8Array[] {
     return v.map(i => this.convertToUint8Array(i));
   }
+
+  private convertMapOrRecord(
+    elements:
+      | Map<string | Uint8Array, string | Uint8Array>
+      | Record<string, string | Uint8Array>
+  ): _DictionaryFieldValuePairGrpc[] {
+    if (elements instanceof Map) {
+      return [...elements.entries()].map(element =>
+        new _DictionaryFieldValuePairGrpc()
+          .setField(this.convert(element[0]))
+          .setValue(this.convert(element[1]))
+      );
+    } else {
+      return Object.entries(elements).map(element =>
+        new _DictionaryFieldValuePairGrpc()
+          .setField(element[0])
```

Comment:
I was doing some inflight troubleshooting here, just FYI. This should be `.setField(this.convert(element[0]))` instead. Looks like changing it back resolves a couple of the test failures as well.